rhel-7.template

```diff
@@ -20,6 +20,7 @@ part / --fstype="ext4" --size=10240
 repo --name=base --baseurl=${base_repo_url}
 repo --name=updates --baseurl=${updates_repo_url}
 repo --name=cdk --baseurl=${cdk_repo_url}
+repo --name=hvkvp --baseurl=https://copr-be.cloud.fedoraproject.org/results/gbraad/go-hvkvp/epel-7-x86_64/
```

Comment:
@gbraad have you tested this, I am afraid if this will work in the brew :( We already started the process to get that package inside the brew but meanwhile, I will have it somewhere so it can pick up by brew.

Reply:
@praveenkumar I can't verify this, and not sure what to do with this. I saw the ticket, but not sure how to progress...

Reply:
@gbraad we already sent mail so can you please remove this from here and that something we can take internally.

Reply:
```diff
-repo --name=hvkvp --baseurl=https://copr-be.cloud.fedoraproject.org/results/gbraad/go-hvkvp/epel-7-x86_64/
+repo --name=hvkvp --baseurl=${hvkvp_repo_url}
```